chain/client/tests/process_blocks.rs

```diff
@@ -2267,6 +2275,135 @@ fn test_epoch_protocol_version_change() {
     assert_eq!(protocol_version, PROTOCOL_VERSION + 1);
 }
 
+#[test]
+#[cfg(feature = "protocol_feature_fix_storage_usage")]
```

Comment:
It is better to group the tests for some protocol feature in one mod so that the imports are cleaner, e.g.

```rust
#[cfg(test)]
mod protocol_feature_fix_storage_usage_tests {
   ....
}